docs/proposals/2019-09-30 - Content by reference in Link Lists.md

```diff
@@ -0,0 +1,64 @@
+## Problem
+Currently, editors can only add URLs in `interstitial_link` to Link Lists. This presents a few issues:
+1. it's not possible to use a Link List until all of the desired content is published
+2. it's not as straightforward to fetch additional information about the story with just the URL
+
+## Proposal
+We propose to add additional traits, `Story`, `Video`, and `Gallery`, to `interstitial_link`, so we can add more types of elements by reference to link lists in addition of `image`.
+
+However, unlink `image`, which is used only for presentation purpose, the `story`, `video`, and `gallery` will be used for a link to it's own content. Therefore, the `url` field should become unavailable when either of the reference to `story`, `video`, and `gallery` are being used.
+## Example
+```JSON
+{
+  "type": "interstitial_link",
+  "subtype": "",
+  "content": "British boxer bites opponent",
+  "description": {
+    "type": "text",
+    "content": "A <em>British boxer</em> has had his license suspended after being disqualified for biting during a <strong>heavyweight</strong> fight."
+  },
+  "oneOf": [{
+    "url": "https://www.washingtonpost.com/sports/2019/04/01/british-boxer-suffers-dental-breakdown-bites-opponent-ring/",
+    "story": {
+      "type": "reference",
+      "referent": {
+        "type": "story",
+        "id": "BBB"
+      }
+    },
+    "video": {
+      "type": "reference",
+      "reference": {
+        "type": "video",
+        "id": "CCC"
+      }
+    },
+    "gallery": {
+      "type": "reference",
+      "reference": {
+        "type": "gallery",
+        "id": "DDD"
+      }
+    },
+  }],
+  "image": {
+    "type": "reference",
+    "referent": {
+      "type": "image",
+      "id": "AAA"
+    }
+  },
+}
+```
+## Questions and Concerns
+**So is the idea that an `interstitial_link` now can link to a url, a story, video or gallery? But only one of those would ever be present?**
+Yes and Yes
+
+**Does this mean “show a gallery here” or “show a link to a gallery here?**
+This means show a link to a gallery here.
+
+**What about the potential performance impact if a content producer adds 20 story reference `interstitial_link`s in a Link List?**
+Link Lists are capped at 10 content elements.
```

Comment:
1. Does capping the list at 10 items raise any concerns over customers soon wanting 12-15 items, etc.
2. I'm a little confused about the concept here - is the idea to add a link to content rather than embedding the content in the story?

Reply:
>Does capping the list at 10 items raise any concerns over customers soon wanting 12-15 items, etc.
It's under discussion, currently the idea is set to 100 or fewer.

>I'm a little confused about the concept here - is the idea to add a link to content rather than embedding the content in the story?
Yes. To be precise, the idea is to add a link (or non-link content, in another PR) to a Link List instead of embedding the content in the story.

Reply:
>Does capping the list at 10 items raise any concerns over customers soon wanting 12-15 items, etc.

It is an ongoing discussion, currently the idea is set to 100 or fewer. 
>I'm a little confused about the concept here - is the idea to add a link to content rather than embedding the content in the story?

Yes, the idea is adding content(video, story, gallery) to link list container (while allowing them to be added by `promo_reference`) rather than embedding the content in the story individually.